aspnetcore/security/enforcing-ssl.md

```diff
@@ -42,7 +42,7 @@ The following code calls [AddHttpsRedirection](/dotnet/api/microsoft.aspnetcore.
 
 The preceding highlighted code:
 
-* Sets [HttpsRedirectionOptions.RedirectStatusCode](/dotnet/api/microsoft.aspnetcore.httpspolicy.httpsredirectionoptions.redirectstatuscode).
+* Sets [HttpsRedirectionOptions.RedirectStatusCode](/dotnet/api/microsoft.aspnetcore.httpspolicy.httpsredirectionoptions.redirectstatuscode) to `Status307TemporaryRedirect`. Production apps may want to use `Status301MovedPermanently`.
```

Comment:
That's the default. Which is fine. I've done this before ... shown the code with the defaults ... but I always call it out. Also, might want to use the corollary for 307, which is 308.

> \* Sets \[HttpsRedirectionOptions.RedirectStatusCode](/dotnet/api/microsoft.aspnetcore.httpspolicy.httpsredirectionoptions.redirectstatuscode) to \`Status307TemporaryRedirect\`, which is the default value. Production apps may want to use \`Status308PermanentRedirect\`.

Note that @Tratcher just said that the 307 is fine for development and production ... the HSTS ` Strict-Transport-Security` will enforce HTTPS, so there's no need to change this for a production app.

Final concern: If it says 'production app may ...', then I wonder if the reader wonders, 'ok ... how do I get the dev version to do one thing and the production version to do another?' ... they possibly need a cross-link to the multiple environments topic.

Reply:
Ditch the 301, refer to HSTS for production.